ui/org.eclipse.pde.core/src/org/eclipse/pde/internal/core/builders/BundleErrorReporter.java

```diff
@@ -553,6 +554,21 @@ private void validateBundleVersion() {
 		}
 	}
 
+	private boolean isCompatibleOsgiEE() {
+		IHeader header = getHeader(Constants.REQUIRE_CAPABILITY);
+		if (header == null) {
+			return false;
+		}
+		String ee = header.getValue();
+		String[] parts = ee.split(";"); //$NON-NLS-1$
+		if (parts[0] == null) {
+			return false;
+		}
+		if (!parts[0].equalsIgnoreCase(osgiEE)) {
+			return false;
+		}
+		return true;
```

Comment:
This is not very robust, in general you probably would want to use `ManifestElement.parseHeader()` instead.
But as written in the general comment, this should be done at a more abstract level.